docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+# Runs SQL LRS with PostgreSQL - Provided for demonstration purposes only!
+# To run: docker compose up
+# See the Docker Compose docs for more info: https://docs.docker.com/compose/
+
+services:
+  db:
+    image: postgres
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: lrsql_user
+      POSTGRES_PASSWORD: lrsql_password
+      POSTGRES_DB: lrsql_db
+  lrs:
+    # build: . # switch to this for active dev
+    image: yetanalytics/lrsql:latest
+    command:
+      - /lrsql/bin/run_postgres.sh
+    ports:
+      - "8080:8080"
+    depends_on:
+      - db
+    environment:
+      LRSQL_API_KEY_DEFAULT: my_key
```

Comment:
Based on talks between Cliff and I, adding `LRSQL_POOL_INITIALIZATION_FAIL_TIMEOUT: 10000` here should suffice for avoiding connection failures.